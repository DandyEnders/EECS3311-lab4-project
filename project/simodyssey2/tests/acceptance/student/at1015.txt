move(N)
move(S)
move(S)
move(N)
move(SW)
move(NW)
move(NW)
test(2,34,64,64,64)
liftoff
move(SW)
move(S)
move(N)
abort
move(NW)
wormhole
