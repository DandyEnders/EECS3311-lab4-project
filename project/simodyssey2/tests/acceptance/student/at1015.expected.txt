  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->move(N)
  state:0.1, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.2, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.3, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.4, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.5, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.6, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.7, error
  Negative on that request:no mission in progress.
->test(2,34,64,64,64)
  state:1.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[1,M],-,-
    [1,2]->[2,M],[3,J],[-10,Y],-
    [1,3]->[4,M],[5,M],-,-
    [1,4]->[6,J],-,-,-
    [1,5]->[-4,W],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[7,M],[-2,W],-,-
    [2,3]->[8,J],[9,J],[-5,W],-
    [2,4]->[10,J],[11,M],-,-
    [2,5]->[12,J],[-8,*],-,-
    [3,1]->[13,M],[14,M],-,-
    [3,2]->[-11,Y],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[15,M],[-7,W],-,-
    [3,5]->[16,M],-,-,-
    [4,1]->[17,M],[18,M],-,-
    [4,2]->[19,J],-,-,-
    [4,3]->[20,M],-,-,-
    [4,4]->-,-,-,-
    [4,5]->-,-,-,-
    [5,1]->[21,M],-,-,-
    [5,2]->[22,J],[23,M],[24,J],[-9,Y]
    [5,3]->[25,M],[-6,W],-,-
    [5,4]->[-3,Y],-,-,-
    [5,5]->[26,M],-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,*]->Luminosity:5
    [-7,W]->
    [-6,W]->
    [-5,W]->
    [-4,W]->
    [-3,Y]->Luminosity:2
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [2,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [3,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [4,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [5,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [6,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [7,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [8,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [9,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [10,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [11,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [12,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [13,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [14,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [15,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [16,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [17,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:2
    [18,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [19,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [20,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [21,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [22,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [23,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [24,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [25,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [26,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EM--   MJY-   MM--   J---   W---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   MW--   JJW-   JM--   J*--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MM--   Y---   O---   MW--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    MM--   J---   M---   ----   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    M---   JMJY   MW--   Y---   M---   
->liftoff
  state:1.1, mode:test, error
  Negative on that request:you are not on a planet at Sector:1:1
->move(SW)
  state:2.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[2,5,3]
    [2,M]:[1,2,1]->[2,1,1]
    [4,M]:[1,3,1]
    [5,M]:[1,3,2]->[5,4,2]
    [6,J]:[1,4,1]->[5,3,3]
    [7,M]:[2,2,1]->[5,4,3]
    [13,M]:[3,1,1]->[2,2,1]
    [15,M]:[3,4,1]->[5,3,4]
    [16,M]:[3,5,1]->[3,1,1]
    [19,J]:[4,2,1]->[4,3,2]
    [20,M]:[4,3,1]->[4,4,1]
    [21,M]:[5,1,1]->[1,1,1]
  Sectors:
    [1,1]->[21,M],[1,M],-,-
    [1,2]->-,[3,J],[-10,Y],-
    [1,3]->[4,M],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[-4,W],-,-,-
    [2,1]->[2,M],-,-,-
    [2,2]->[13,M],[-2,W],-,-
    [2,3]->[8,J],[9,J],[-5,W],-
    [2,4]->[10,J],[11,M],-,-
    [2,5]->[12,J],[-8,*],[0,E],-
    [3,1]->[16,M],[14,M],-,-
    [3,2]->[-11,Y],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[-7,W],-,-
    [3,5]->-,-,-,-
    [4,1]->[17,M],[18,M],-,-
    [4,2]->-,-,-,-
    [4,3]->-,[19,J],-,-
    [4,4]->[20,M],-,-,-
    [4,5]->-,-,-,-
    [5,1]->-,-,-,-
    [5,2]->[22,J],[23,M],[24,J],[-9,Y]
    [5,3]->[25,M],[-6,W],[6,J],[15,M]
    [5,4]->[-3,Y],[5,M],[7,M],-
    [5,5]->[26,M],-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,*]->Luminosity:5
    [-7,W]->
    [-6,W]->
    [-5,W]->
    [-4,W]->
    [-3,Y]->Luminosity:2
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [2,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [3,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [4,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:1
    [5,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:1
    [6,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:2
    [7,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:2
    [8,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [9,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [10,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [11,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [12,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [13,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [14,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [15,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:1
    [16,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [17,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [18,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [19,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:2
    [20,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:2
    [21,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [22,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [23,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [24,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [25,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [26,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    MM--   -JY-   M---   ----   W---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    M---   MW--   JJW-   JM--   J*E-   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MM--   Y---   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    MM--   ----   -J--   M---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   JMJY   MWJM   YMM-   M---   
->move(S)
  state:3.0, mode:test, ok
  Movement:
    [0,E]:[2,5,3]->[3,5,1]
    [1,M]:[1,1,2]->[1,5,2]
    [2,M]:[2,1,1]->[3,2,2]
      reproduced [27,M] at [3,2,3]
    [3,J]:[1,2,2]->[2,2,3]
    [10,J]:[2,4,1]->[2,3,4]
    [11,M]:[2,4,2]
    [13,M]:[2,2,1]->[5,5,2]
      reproduced [28,M] at [5,5,3]
    [14,M]:[3,1,2]->[4,1,3]
    [16,M]:[3,1,1]->[3,5,2]
      reproduced [29,M] at [3,5,3]
      attacked [0,E] at [3,5,1]
    [18,M]:[4,1,2]->[4,2,1]
    [23,M]:[5,2,2]->[1,1,2]
    [25,M]:[5,3,1]->[5,1,1]
  Sectors:
    [1,1]->[21,M],[23,M],-,-
    [1,2]->-,-,[-10,Y],-
    [1,3]->[4,M],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[-4,W],[1,M],-,-
    [2,1]->-,-,-,-
    [2,2]->-,[-2,W],[3,J],-
    [2,3]->[8,J],[9,J],[-5,W],[10,J]
    [2,4]->-,[11,M],-,-
    [2,5]->[12,J],[-8,*],-,-
    [3,1]->-,-,-,-
    [3,2]->[-11,Y],[2,M],[27,M],-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[-7,W],-,-
    [3,5]->[0,E],[16,M],[29,M],-
    [4,1]->[17,M],-,[14,M],-
    [4,2]->[18,M],-,-,-
    [4,3]->-,[19,J],-,-
    [4,4]->[20,M],-,-,-
    [4,5]->-,-,-,-
    [5,1]->[25,M],-,-,-
    [5,2]->[22,J],-,[24,J],[-9,Y]
    [5,3]->-,[-6,W],[6,J],[15,M]
    [5,4]->[-3,Y],[5,M],[7,M],-
    [5,5]->[26,M],[13,M],[28,M],-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,*]->Luminosity:5
    [-7,W]->
    [-6,W]->
    [-5,W]->
    [-4,W]->
    [-3,Y]->Luminosity:2
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:2/3, life:2/3, landed?:F
    [1,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:2
    [2,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:2
    [3,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:0
    [4,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:0
    [5,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:0
    [6,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [7,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:1
    [8,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [9,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [10,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [11,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:2
    [12,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [13,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:2
    [14,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:2
    [15,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:0
    [16,M]->fuel:1/3, actions_left_until_reproduction:1/1, turns_left:1
    [17,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [18,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:2
    [19,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [20,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [21,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [22,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [23,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [24,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [25,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:0
    [26,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [27,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [28,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [29,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    MM--   --Y-   M---   ----   WM--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   -WJ-   JJWJ   -M--   J*--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   YMM-   O---   -W--   EMM-   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    M-M-   M---   -J--   M---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    M---   J-JY   -WJM   YMM-   MMM-   
->move(N)
  state:4.0, mode:test, ok
  Movement:
    [0,E]:[3,5,1]->[2,5,3]
    [3,J]:[2,2,3]->[1,2,1]
    [4,M]:[1,3,1]->[5,3,1]
    [5,M]:[5,4,2]->[1,4,1]
      reproduced [30,M] at [1,4,2]
    [8,J]:[2,3,1]->[3,2,4]
    [9,J]:[2,3,2]->[2,2,1]
    [12,J]:[2,5,1]->[3,1,1]
    [15,M]:[5,3,4]->[5,4,2]
      reproduced [31,M] at [5,4,4]
    [17,M]:[4,1,1]->[4,2,2]
    [21,M]:[1,1,1]->[5,1,2]
      reproduced [32,M] at [5,1,3]
    [22,J]:[5,2,1]->[4,2,3]
    [24,J]:[5,2,3]->[5,1,4]
    [25,M]:[5,1,1]->[4,1,1]
      reproduced [33,M] at [4,1,2]
    [26,M]:[5,5,1]->[1,4,3]
  Sectors:
    [1,1]->-,[23,M],-,-
    [1,2]->[3,J],-,[-10,Y],-
    [1,3]->-,-,-,-
    [1,4]->[5,M],[30,M],[26,M],-
    [1,5]->[-4,W],[1,M],-,-
    [2,1]->-,-,-,-
    [2,2]->[9,J],[-2,W],-,-
    [2,3]->-,-,[-5,W],[10,J]
    [2,4]->-,[11,M],-,-
    [2,5]->-,[-8,*],[0,E],-
    [3,1]->[12,J],-,-,-
    [3,2]->[-11,Y],[2,M],[27,M],[8,J]
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[-7,W],-,-
    [3,5]->-,[16,M],[29,M],-
    [4,1]->[25,M],[33,M],[14,M],-
    [4,2]->[18,M],[17,M],[22,J],-
    [4,3]->-,[19,J],-,-
    [4,4]->[20,M],-,-,-
    [4,5]->-,-,-,-
    [5,1]->-,[21,M],[32,M],[24,J]
    [5,2]->-,-,-,[-9,Y]
    [5,3]->[4,M],[-6,W],[6,J],-
    [5,4]->[-3,Y],[15,M],[7,M],[31,M]
    [5,5]->-,[13,M],[28,M],-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,*]->Luminosity:5
    [-7,W]->
    [-6,W]->
    [-5,W]->
    [-4,W]->
    [-3,Y]->Luminosity:2
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:2/3, landed?:F
    [1,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [2,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [3,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:0/2, turns_left:1
    [4,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [5,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:2
    [6,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:0
    [7,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:0
    [8,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [9,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [10,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:0
    [11,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:1
    [12,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:0
    [13,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:1
    [14,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [15,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [16,M]->fuel:1/3, actions_left_until_reproduction:1/1, turns_left:0
    [17,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [18,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [19,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:0
    [20,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [21,M]->fuel:1/3, actions_left_until_reproduction:1/1, turns_left:1
    [22,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [23,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [24,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [25,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:2
    [26,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [27,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [28,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [29,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [30,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [31,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [32,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [33,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -M--   J-Y-   ----   MMM-   WM--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   JW--   --WJ   -M--   -*E-   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    J---   YMMJ   O---   -W--   -MM-   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    MMM-   MMJ-   -J--   M---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -MMJ   ---Y   MWJ-   YMMM   -MM-   
->abort
  state:4.1, ok
  Mission aborted. Try test(3,5,7,15,30)
->move(NW)
  state:4.2, error
  Negative on that request:no mission in progress.
->wormhole
  state:4.3, error
  Negative on that request:no mission in progress.
